Client/Frontend/Browser/Certificate Viewer/CertificateViewController.swift

```diff
@@ -0,0 +1,527 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import BraveCore
+import SwiftUI
+import BraveUI
+
+class BraveCertificate: ObservableObject {
+    @Published var value: BraveCertificateModel
```

Comment:
`value` is never mutated so why is this an `ObservableObject`/`Published`?

Reply:
This whole class should probably be removed, add extensions instead to `BraveCertificateModel` so that it can be created with those alternative datasets